README.md

```diff
@@ -129,14 +129,14 @@ osdctl account clean-velero-snapshots -p <profile name> -c <config file path>
 
 ### AWS Account IAM User Credentials validation
 
-`check-secrets` command checks the IAM User Secret associated with Account Accout CR.
+`verify-secrets` command verifies the IAM User Secret associated with Account Accout CR.
```

Comment:
This change makes sense, but it might break SOP and people might be used to the older version. Also, we will need a major version bump to `osdctl` after the rename. 

Unless it's not a hard requirement from a ticket, I would recommend leaving it as is and only keep the flag feature. 
Feel free to open a separate PR and discuss it with other folks too so we can move forward with the rename.